x-pack/plugins/fleet/public/components/platform_selector.tsx

```diff
@@ -37,19 +43,37 @@ const CommandCode = styled.pre({
   overflow: 'auto',
 });
 
-const K8S_COMMAND = `kubectl apply -f elastic-agent-managed-kubernetes.yaml`;
-
 export const PlatformSelector: React.FunctionComponent<Props> = ({
   linuxCommand,
   macCommand,
   windowsCommand,
   linuxDebCommand,
   linuxRpmCommand,
+  k8sCommand,
   isK8s,
```

Comment:
I think that keeping the name `isK8s` for this prop is now a bit confusing, if I understand correctly it now only indicates the presence of the kubernetes package in the policy, while the kubernetes platform is always present.
I would change it to something like `hasK8sIntegration` or something like that.